easybuild/easyblocks/g/gromacs.py

```diff
@@ -70,6 +70,26 @@ def __init__(self, *args, **kwargs):
         self.lib_subdir = ''
         self.pre_env = ''
 
+        # check whether PLUMED is listed as a dependency
+        self.with_plumed = 'PLUMED' in [dep['name'] for dep in self.cfg['dependencies']]
```

Comment:
why not just check this in `prepare_step`, since `self.with_plumed` isn't used anywhere else?

also, we usually check this via `get_software_root('PLUMED')` (even though that's not perfect, see https://github.com/hpcugent/easybuild-framework/issues/1269)